duden/display.py

```diff
@@ -152,3 +152,6 @@ def describe_word(word):
         for part_of_speech, words in word.compounds.items():
             print(blue(' - {}:'.format(part_of_speech.capitalize())),
                   ', '.join(words))
+
+    if word.alternative_spellings:
+        print(white(_('Alternative_spellings:'), bold=True), word.alternative_spellings)
```

Comment:
This is supposed to be human-readable string, so `'Alternative spellings:'` without underscore should be better. Also could you add a newline at the end of the file?